submissions/Akshata-4-2021-other/Akshata-4-2021-other.md

```diff
@@ -1 +1,49 @@
+## Opscientia Open Web Fellowship: A platform to store preprints and publications permanently 
+name: Akshata, email: makshata403@gmail.com, timezone: CEST, discordID: Akshata M#6695, current role: kernel fellow (KB4)
+
+### Overview
+
+The greatest treasure mankind can save for its future generations is knowledge. Knowledge in any form should be secured and never be lost. This project focuses on scientific knowledge of any form including research papers, preprints, writeups and thesis which could be open source. It will be built on 'Arweave' which assures that the papers uploaded on the platform will never be lost (would stay for thousands of years without any additional costs) and it is decentralized. 
```

Comment:
can you please provide a reference for ARWEAVE? i think current models support 200 years of archival on their block state mechanism, a simple explanation of how this works would help blockchain naive reviewers assess the feasibility in more detail